src/cli/cli_commissioner.cpp

```diff
@@ -351,6 +532,20 @@ template <> otError Commissioner::Process<Cmd("id")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli commissioner start
+ * @code
+ * commissioner start
+ * Commissioner: petitioning
+ * Done
+ * Commissioner: active
+ * @endcode
+ * @par
+ * Enables the Thread %Commissioner role.
+ * @par
+ * This command causes the device to send `LEAD_PET` and `LEAD_KA` messages.
```

Comment:
```suggestion
 * This command enables the Thread Commissioner role.
```
Suggest changing this line since the msgs sent are part of the internal behavior of a commissioner.